### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by ubaldot** [mathworks/MATLAB-language-server#51] [Q&A]
I want to try it with Vim9 and yegappan/lsp client. 

However, I cannot see how install it. It looks like the only way is to install nodejs and then install it through npm, but it would be great to write a section in the readme. 

Also, wouldn't be possible to distribute binaries through channels like e.g. conda-forge? 
Finally, is there any CLI API documented somewhere? 

I wish to use it in WSL but I think that I should specify the actual Matlab path somehow. 

Thanks!

**Comment by kurokirasama** [comment]
hey, i compiled instructions in how to install here: https://github.com/kurokirasama/matlab_lsp_sublime

I dont know specifics for vscode though
